pkg/operator/ceph/file/mds/spec.go

```diff
@@ -132,7 +132,7 @@ func (c *Cluster) makeMdsDaemonContainer(mdsConfig *mdsConfig) v1.Container {
 		"--foreground",
 	)
 
-	if !c.clusterSpec.Network.IsHost() {
+	if !c.clusterSpec.Network.IsHost() && !c.clusterSpec.Network.IsMultus() {
```

Comment:
maybe a comment to it why the --pubic-addr is not required

Reply:
Do you want to a specific message? since I think the condition is

Reply:
Yaa condition looks fine,
I mean to tell developer why we wont need that

Reply:
Yaa condition looks fine,
I mean to tell developer why we wont need that